orion/pkg/controllers/announce_controller_test.go

```diff
@@ -186,7 +186,7 @@ func TestDeleteAnnounce_Success(t *testing.T) {
 	services.AnnounceService = &mas
 
 	// Create new HTTP request to endpoint
-	recorder := sendHttpRequest(t, http.MethodDelete, "/api/announcements/v1/announce/1", nil)
+	recorder := sendHttpRequest(t, http.MethodDelete, "/api/announcements/v1/announcement/1", nil)
```

Comment:
If you're going to change this, make sure it is accurately reflected in `handler.go`